### DISCUSSION TITLE
Some effects of the entry animation may have abnormal hints

### DISCUSSION THREAD

**Discussion by pshow3** [qrpcode/pptshow#4] [Q&A] [answered]
Some of the entrance animations may have some exceptions, such as the time is particularly fast or the display effect and Office inconsistent situation

**Comment by qrpcode** [comment, marked as answer]
It is true that some animations are buggy, we recommend users to try the tested animations first to make sure they work. We are still working on other animations!
Available animations reference files: https://pptshow.cc/book/InAnimation.html
